client/my-sites/site-settings/navigation-back-button/README.md

```diff
@@ -0,0 +1,41 @@
+*Navigation Back Button*
+
+This component is used to display a `Back` button for redirection to the previously
+visited site.
+
+#Usage:
+
+```js
+import NavigationBackButton from 'my-sites/site-settings/navigation-back-button';
+...
+
+getRoute() {
```

Comment:
This one is probably not necessary. Since we try to keep examples as simple as possible, we can just include a dummy path here instead of a method.